src/background/deinflect.test.ts

```diff
@@ -309,7 +315,150 @@ describe('deinflect', () => {
     const result = deinflect('食べて');
     const match = result.find((candidate) => candidate.word == '食べる');
     expect(match).toBeDefined();
-    expect(match!.reasons).not.toContainEqual([Reason.Continuous]);
+    expect(match!.reasonChains).not.toContainEqual([Reason.Continuous]);
+  });
+
+  it('deinflects respectful continuous forms', () => {
+    const cases: Array<[string, string, Array<Reason>, Array<Reason>]> = [
+      ['行っていらっしゃる', '行く', [], []],
```

Comment:
I think these tests might be easier to read if we explicitly list out the reason chains for each case. It's a bit confusing that the first chain gets extra reasons appended to it but the second one doesn't.

Similarly for the other tests, unless all the tests expect the same reasons, I think it's best to list the expected reasons along with each case.

Reply:
Sure, will do. Personally, I find it harder to read when line wrapping occurs, so if it's okay, I'll add prettier-ignore again.